installer/helm/chart/volcano/config/volcano-scheduler-ci.conf

```diff
@@ -2,6 +2,7 @@ actions: "enqueue, allocate, backfill, reclaim, preempt"
 tiers:
 - plugins:
   - name: priority
+    enableReclaimable: false
```

Comment:
How about set this false default？

Reply:
how to？Add a param for priority plugin and default not to register  `AddReclaimableFn`?

Reply:
It already set `enableReclaimable` as false in installing yaml.